src/browser-realm.ts

```diff
@@ -92,6 +95,11 @@ export default function createSecureEnvironment(options?: BrowserEnvironmentOpti
     // to detach the iframe only if the keepAlive option isn't true
     if (keepAlive !== true) {
         removeIframe(iframe);
+    } else {
+        // TODO: temporary hack to preserve the document reference in FF
+        // https://bugzilla.mozilla.org/show_bug.cgi?id=543435
+        open.call(redRefs.document);
+        close.call(redRefs.document);
```

Comment:
it is fine to call open or closed from blue realm with the ref to the red realm, that works just fine! we can't use the one from red realm because it was remapped to the blue one.

Reply:
☝️ where is it that calling it directly wouldn't work right?

Reply:
@jdalton I didn't understand your question.

Reply:
Never mind. I see now.